### PR TITLE
Added Header Section

### PR DESCRIPTION
**Pull request by emmyobonyo**
### Tasks Completed
- Added display picture.
- Added profile details.
- Added relevant buttons.

#### I had a tough time with the buttons hence the long time. And also it was my first time working with Figma, so I had to learn how to be more detail-oriented., hence more time, but I'm learning.

### CODE REVIEW

**Comment by emmyobonyo** [comment]
- I realized I should've created a new branch for the 'About section.' I guess it skipped my mind till it was a bit late.

File: README.md [hunk 1]
```diff
@@ -1,70 +1,41 @@
...
+![Home Page](./public/home.png)
 
-In the project directory, you can run:
+- This is the Header section of the App
 
-### `yarn start`
+## Built With
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
```
Comment (KnightWorker37) [inline README.md:12]: Not a bad idea to leave this in

File: public/index.html [hunk 2]
```diff
@@ -2,15 +2,15 @@
...
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <meta name="theme-color" content="#000000" />
     <meta
       name="description"
       content="Web site created using create-react-app"
     />
-    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
```
Comment (KnightWorker37) [inline public/index.html:12]: It was not necessary to delete this stuff. Actually going forward it's going to be kind of essential. Please read a bit more about PWA:

https://developer.mozilla.org/en-US/docs/Web/Progressive_web_apps
https://web.dev/progressive-web-apps/

File: src/App.js [hunk 3]
```diff
@@ -1,9 +1,12 @@
 import './App.css';
+import Header from './components/Header';
+import About from './components/about/About';
```
Comment (KnightWorker37) [inline src/App.js:3]: Looking at the official recommendations: https://reactjs.org/docs/faq-structure.html

I guess it should be not be that nested? So either:

```suggestion
import Header from './header/Header';
import About from './about/About';
```
or:
```suggestion
import Header from './components/Header';
import About from './components/About';
```

What do you think?

File: src/components/Header.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import './Header.css';
+
+function Header() {
+  return (
+    <header>
+      <img src="./obonyo.jpeg" alt="Emmanuel Obonyo" />
```
Comment (KnightWorker37) [inline src/components/Header.js:7]: It would be nice if all this displayed data could be extracted to input arguments to make it reusable?

**Review by KnightWorker37 — CHANGES_REQUESTED** [review]
And finally you can also include the editable Figma doc in case anyone wants to edit that in future? Is that even possible?

**Comment by raybesiga** [comment]
I believe that it is possible to share a Figma link @KnightWorker37 @emmyobonyo

File: src/App.js (hunk 3 above)
Comment (emmyobonyo) [reply src/App.js:3]: Agreed. Thanks for the suggestion. Too much nesting is definitely not good in the long run.

File: README.md (hunk 1 above)
Comment (emmyobonyo) [reply README.md:12]: - Done ✅

File: public/index.html (hunk 2 above)
Comment (emmyobonyo) [reply public/index.html:12]: - I've browsed through the documentation and it's actually very important. I'm going to start leaving them in my future apps.

File: src/components/Header.js (hunk 4 above)
Comment (emmyobonyo) [reply src/components/Header.js:7]: Noted.

**Pull request merged**